WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -228,6 +252,49 @@ private extension OrdersRootViewController {
         }
     }
 
+    func configureUpsellCardReaderFeatureAnnouncement() {
+        guard upsellCardReadersAnnouncementViewModel.shouldBeVisible else {
+            return
+        }
+
+        let view = FeatureAnnouncementCardView(viewModel: upsellCardReadersAnnouncementViewModel,
+                                               dismiss: { [weak self] in
+            self?.hideUpsellCardReaderFeatureAnnouncementView(true)
+        }, callToAction: {
+            let configuration = CardPresentConfigurationLoader().configuration
+            WebviewHelper.launch(configuration.purchaseCardReaderUrl(), with: self)
```

Comment:
In this case, doesn't change much, but we also have a new [SafariView SwiftUI component](https://github.com/woocommerce/woocommerce-ios/blob/6febf756b7de07d7abf17d43cd4243dfd590f37f/WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person%20Payments/SafariView.swift) we can use  👯

Reply:
Good point! The problem here is that we are in `UIKit` context, so we cannot use the `SwiftUI` component